app/facades/chores_facade.rb

```diff
@@ -0,0 +1,7 @@
+class ChoresFacade
+  def self.create_chore(chore_info)
+    chore_attributes = ChoreService.create_or_find_chore(chore_info)
+    formatted = chore_attributes[:data][:attributes]
+    Chore.new(formatted)
```

Comment:
I'm not sure that we can call chore.new here since we don't have a database in the frontend. Did this pass in your test?

Reply:
The 'chore' related files are not done, and are works in-progress as a part of the new chore setup.  They do not have tests yet.  

This 'dashboards' branch is only intended to incorporate the finished sessions/oauth setup.  I could take all the chore stuff out for this PR, but would just have to put the chore stuff back in when I continue work on new chore.

I also noticed the Travis errors, some went back to the household setup it looks like.